src/project/how-to-contribute.md

```diff
@@ -1,5 +1,67 @@
 # How to contribute
 
+First of all, happy that you are here on this page and thinking about contributing to the project.
+
+**The Vue.js community thanks you.**
+
+## Development
+
+To contribute to the development of the site, you can initially open an issue on Github and report an error or propose a new feature.
+
+If you are willing to develop, there is no mystery, just open a Pull Request following the template.
+
+This website uses Vuepress, TailwindCSS, and Composition API. 
+
+::: alert tip
+Try to follow the official style guide for Vue.js as much as possible.
+:::
+
+## Writing
+
+We will love to hear your content idea, so the first step is to open an issue using the specific template.
+
+You can open an issue:
+
+- [New Post/Article](https://github.com/vue-a11y/vue-a11y.com/issues/new?labels=Post&template=new-post.md&title=Post%2FArticle%3A+%5BYour+title%5D)
+- [New Widget (Recipe)](https://github.com/vue-a11y/vue-a11y.com/issues/new/choose)
+- [New Tip (Recipe)](https://github.com/vue-a11y/vue-a11y.com/issues/new/choose)
+
+We also have an [issue](https://github.com/vue-a11y/vue-a11y.com/issues/14) with several content ideas, feel free to add other subjects that you want to see as content on the site.
+
+Our intention is to reward the creator for each content published on the site, we will soon achieve this feat.
+
+## Answering questions
+
+In our [GitHub Problem Tracker](https://github.com/vue-a11y/vue-a11y.com/issues), we are open to answering questions from people of all levels of knowledge.
+
+We will be grateful to receive your help by answering questions and also sharing your experience with us.
+
+We are also daily answering questions in the Vue.js community on Discord ([Vue Land](https://discord.gg/vue)), join us.
+
+## Sharing
+
+When you share, you are helping the web community to understand more about accessibility through content.
+
+[Vue a11y](/) focuses specifically on helping the Vue.js community by creating content, plugins, and components that make it easier on a daily basis.
+
+We count on your support and the support of the community to make it reach your friends and other developers.
+
+## Supporting
+
+The project is supported by people who love Vue.js and accessibility, and who know the importance of the web being more inclusive.
+
+Your support is very important so that we can devote more time to creating content, keeping plugins and components up to date, in addition to funding content creators for their time dedicated to creating robust and well-tested posts.
+
+<p>
+We have separated some rewards for our Open Collective supporters.
+<br>
+<a href="https://opencollective.com/vue-a11y/donate" target="_blank" rel="noopener" style="background-color: initial; width: 300px;">
+  <img src="https://opencollective.com/vue-a11y/donate/button@2x.png?color=blue" style="box-shadow: none; display: inline;" />
```

Comment:
Needs alt

Reply:
🤦‍♂️ I copied/pasted and forgot to check.

Thanks @marcus-herrmann